src/boot/bootctl-install.c

```diff
@@ -319,13 +319,14 @@ static int create_subdirs(const char *root, const char * const *subdirs) {
         return 0;
 }
 
-static int update_efi_boot_binaries(const char *esp_path, const char *source_path) {
+static int update_efi_boot_binaries(const char *esp_path, const char *source_path, const char *exclude_path) {
```

Comment:
BTW, I think we should actually handle up-to-date binaries gracefully here. These aren't canonical paths that `bootctl install` would operate on after all, and if users copy other versions of sd-boot here for testing/other purposes we shouldn't error out.